corehq/apps/users/tests/test_util.py

```diff
@@ -260,3 +263,46 @@ def test_returns_unchanged_username_if_already_complete(self):
     def test_returns_complete_username_if_incomplete(self):
         username = get_complete_mobile_username('test', 'test-domain')
         self.assertEqual(username, 'test@test-domain.commcarehq.org')
+
+
+class TestGetLocationsWithSingleUser(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain = 'test-domain'
+        cls.user_id = 'test-user-id'
+        cls.location_ids = ['123']
+        locations = [
+            SQLLocation(
+                name='Brazil',
+                location_id='123'
+            ),
+            SQLLocation(
+                name='Asia',
+                location_id='456'
+            )
+        ]
+
+        get_locations_patch = patch('corehq.apps.users.views.utils._get_location_case_counts_with_single_user')
```

Comment:
Hmm, feels a bit weird mocking this out. I would vote to not do this in order to get more coverage, or add a separate test that verifies that this function returns what we expect it to.

Reply:
Talked with Zandre and decided it's maybe a bit overkill here